<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptWindow</class>
 <widget class="QDialog" name="ReceiptWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonYes">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Kyll√§</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="buttonNo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Ei</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tulostetaanko Kuitti?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
